bootstrap.sh

```diff
@@ -26,7 +26,7 @@ else
 fi
 
 mvn_install() {
-    mvn --batch-mode install -Dmaven.test.skip=true -Dmaven.javadoc.skip=true "$@"
+    mvn --batch-mode --threads 1C -nsu install -Dmaven.test.skip=true -Dmaven.javadoc.skip=true "$@"
```

Comment:
I have no idea - usually it's a bit over 1C and 25% over is fairly conservative. Consider this for later then.